core/src/test/java/apoc/cypher/CypherIsVersionDifferentTest.java

```diff
@@ -12,30 +10,33 @@ public class CypherIsVersionDifferentTest {
 
     @Test
     public void shouldReturnFalseOnlyWithCompatibleVersion() {
-        assertTrue(isVersionDifferent(List.of("3.5"), "4.4.0.2"));
-        assertTrue(isVersionDifferent(List.of("5_0"), "4.4.0.2"));
-        assertFalse(isVersionDifferent(List.of("3.5.12"), "3.5.0.9"));
-        assertFalse(isVersionDifferent(List.of("3.5.12"), "3.5.1.9"));
-        assertFalse(isVersionDifferent(List.of("4.4.5"), "4.4.0.4"));
-        assertFalse(isVersionDifferent(List.of("5.1.1"), "5.1.0"));
-        assertTrue(isVersionDifferent(List.of("5.1.0"), "5.2.0"));
+        assertTrue(isVersionDifferent("3.5", "4.4.0.2"));
+        assertTrue(isVersionDifferent("5_0", "4.4.0.2"));
+        assertFalse(isVersionDifferent("3.5.12", "3.5.0.9"));
+        assertFalse(isVersionDifferent("3.5.12", "3.5.1.9"));
+        assertFalse(isVersionDifferent("4.4.5", "4.4.0.4"));
+        assertFalse(isVersionDifferent("5.1.1", "5.1.0"));
+        assertTrue(isVersionDifferent("5.1.0", "5.2.0"));
+        assertFalse(isVersionDifferent("5.1-aura", "5.1.0"));
+        assertTrue(isVersionDifferent("4.4-aura", "5.1.0"));
```

Comment:
Is that correct? :sweat_smile:

Reply:
Yes I think so. The method checks if the versions are **not** compatible. So this checks that Neo4j 5.1.x and 5.1-aura are compatible with apoc 5.1.x while the combos Neo4j 5.1.x/APOC 5.2.x resp. Neo4j 4.4-aura/APOC 5.1.x are not compatible

Reply:
I completely missed the assertTrue/asertFalse difference. Thank you for the explanation :+1: